CharacterController2D.cs

```diff
@@ -64,7 +64,7 @@ private void FixedUpdate()
 	public void Move(float move, bool crouch, bool jump)
 	{
 		// If crouching, check to see if the character can stand up
-		if (!crouch)
+		if (crouch)
```

Comment:
The soultion brings a good fix . Please accept the pull request.

Reply:
Had a similar solution. @ATBrackeys this is a bug and should be addressed.

Reply:
Looks like it was intended to check when the Parameter for crouch was false (the user/player stop holding the crouch input button) so then it could check if the PlayerObject could "stand up" 

Still learning unity but adding something below  as well as setting `k_CeilingRadius` to a lower float fixed my problem
```suggestion
		if (m_wasCrouching && !crouch)
```